docs/training-apis/experiment-config.txt

```diff
@@ -83,10 +83,12 @@ example:
    preemption, in the unit of batches. The number of records in a batch is controlled by the
    :ref:`global_batch_size <config-global-batch-size>` hyperparameter. Defaults to ``100``.
 
-   -  Setting this to be a small number can reduce the overhead of system operations.
-   -  Setting this to be too large might prevent the system to preempt the resources of this trial
-      to run other tasks.
-   -  As a rule of thumb, it should be set to make each step take 60--180 seconds.
+   -  Setting this value too small can increase the overhead of system operations and decrease
+      training throughput.
+   -  Setting this value too large might prevent the system from reallocating resources from this
+      workload to another, potentially more important, workload.
+   -  As a rule of thumb, it should be set to the number of batches that can be trained in roughly
```

Comment:
Is this rule of thumb still true? @rb-determined-ai may have more context here since scheduling unit is purely a python side detail now.

Reply:
This is fine for now.  It will be ignored by Core API experiments, but given the timeline on which we need to land Core API there's going to be quite a few fields like that for a little while, unfortunately.

Some day, when we have Trainer APIs, this is a field that should just get rm-rf'd from the experiment config.  In the future the experiment config should have three kinds of fields: fields needed for the master to start training, a structured hparams field, and an opaque user_data field (presently called `data`).  The experiment config in the long term, won't include anything that is a training-loop detail; those should all get configured in the python code that invokes the trainer api.